student.rb

```diff
@@ -1,6 +1,8 @@
 require_relative 'person'
 
 class Student < Person
+  attr_accessor :classroom
```

Comment:
* Kindly add a method that serves as the 'belongs-to' side (a student belongs to a classroom) Link to the article with examples: https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships  
*  [Check requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_associations.md#:~:text=Create%20the%20belongs%2Dto%20side%20(a%20student%20belongs%20to%20a%20classroom).)